### DISCUSSION TITLE
Extruder stops after 5-10 layers

### DISCUSSION THREAD

**Discussion by Fallstern** [1coderookie/Klipper4KobraGoNeo#16] [Q&A]
Hello,
I seem to have encountered a problem while printing. My extruder stops working in the middle of the process, and when I try to extrude using the console command, nothing happens. The only way to get it back to working is to restart Klipper. 

Has anyone else faced this issue before, and if so, could you please share any possible solutions? Any suggestions on how to fix it will be greatly appreciated.

**Comment by 1coderookie** [comment]
Hi,
which printer? Go or Neo?  
And do you have to restart the Klipper firmware of your printer then, or the host that's running Klipper?  

It *could* be that you're suffering from a breaking/broken extruder wiring, especially when it's the Neo: https://1coderookie.github.io/KobraGoNeoInsights/hardware/printhead/#extruder-motor-wiring

**Reply by Fallstern** [reply]
Its Kobra Go
I need to restart firmware of my printer.

It's not look like wiring problem
I found that some guys had the same problem on stock firmware too:
https://www.reddit.com/r/anycubic/comments/17jzswj/extruder_stopping_mid_print_on_kobra_printer/

**Reply by 1coderookie** [reply]
Hmm - is your extruder motor *hot* when that happens?  
And when does that happen? Like instantly, after 10min, after 3hrs? 

As for that reddit post: originally that's about the Kobra, which uses a different mainboard, but I saw that one guy with a Kobra Go and Klipper yuo replied to which had the same issue..

**Reply by Fallstern** [reply]
Its hard to catch the moment when it just stops printing. And the time when it stops is random. Some prints just go to finish without this bug.

**Reply by 1coderookie** [reply]
And what about the motor temp? When it occurs and you touch it, does it feel like it's 'unhealthy' hot?

**Reply by Fallstern** [reply]
It's cold, but i need touch it in 5 minutes after motor stopped to figure it is hot.

**Reply by 1coderookie** [reply]
I don't get what you mean..  
If it's too hot, it might be that you have either a faulty motor or the Vref of the stepper driver is set too high. But if you didn't change it and never had issues before, then I'd say it's not the latter one.   
Maybe you have a problem with the mainboard and the stepper driver as well, I don't know. 
However, it doesn't sound like it's an issue initially caused by Klipper. Of course you can try and flash the latest version, but I actually don't expect that would change anything. But since it's not much of an effort, maybe do that anyway.    
If you come across the actual problem that's causing this, please let me know then.

**Reply by Fallstern** [reply]
It's not the cable I've changed it. it's not the motor changed it too. The last thing to check is motherboard. Maybe I will replace it with BTT SKR

**Comment by Fallstern** [comment]
To end this discussion:
I replaced the motherboard with BTT SKR e3 v3.0 and problem with the extruder disappeared

**Reply by 1coderookie** [reply]
Thanks for the feedback, glad you fixed it!
